static/js/services/screensharing.js

```diff
@@ -53,6 +70,29 @@ define(['underscore', 'text!partials/screensharedialogff.html', 'webrtc.adapter'
 			// Define our helpers.
 			this.prepare = null;
 			this.cancel = null;
+			var that = this;
+
+			var selectFirefoxScreenToShare = function(options) {
```

Comment:
`that` is used throughout the `initialize` function. See lines [213](https://github.com/theurere/spreed-webrtc/blob/add-firefoxextension-service/static/js/services/screensharing.js#L213) and [220](https://github.com/theurere/spreed-webrtc/blob/add-firefoxextension-service/static/js/services/screensharing.js#L220). The initialize function should really be broken up.